include/oneapi/dpl/internal/reduce_by_segment_impl.h

```diff
@@ -78,13 +78,15 @@ class Reduce3;
 template <typename Name>
 class Reduce4;
 
-template <typename Policy, typename InputIterator1, typename InputIterator2, typename OutputIterator1,
+template <class _Tag, typename Policy, typename InputIterator1, typename InputIterator2, typename OutputIterator1,
           typename OutputIterator2, typename BinaryPred, typename BinaryOperator>
-oneapi::dpl::__internal::__enable_if_host_execution_policy<Policy, ::std::pair<OutputIterator1, OutputIterator2>>
-reduce_by_segment_impl(Policy&& policy, InputIterator1 first1, InputIterator1 last1, InputIterator2 first2,
+::std::pair<OutputIterator1, OutputIterator2>
+reduce_by_segment_impl(_Tag, Policy&& policy, InputIterator1 first1, InputIterator1 last1, InputIterator2 first2,
                        OutputIterator1 result1, OutputIterator2 result2, BinaryPred binary_pred,
                        BinaryOperator binary_op)
 {
+    static_assert(__internal::__is_backend_tag_v<_Tag>);
```

Comment:
What does `__internal::__is_backend_tag` trait mean?
Probably, should be `__is_backend_tag_serial_v` instead?

Reply:
Agree. Fixed.